includes/forms/class-llms-form-handler.php

```diff
@@ -319,8 +319,11 @@ public function submit( $posted_data, $location, $args = array() ) {
 		// Determine the user action to perform.
 		$action = get_current_user_id() ? 'update' : 'registration';
 
-		// Load the form.
+		// Load the form, filtering out invisible fields.
+		add_filter( 'llms_filter_out_invisible_field', '__return_true', 999 );
```

Comment:
Change explained below.

```suggestion
		add_filter( 'llms_forms_remove_invisible_field', '__return_true', 999 );
```